src/components/bookComponent.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+
+const Book = (props) => (
+  <div>
+    <div>
+      <h6>{props.type}</h6>
+      <h3>{props.name}</h3>
+      <h6>{props.author}</h6>
+      <ul>
+        <li>Comments</li>
+        <li>Remove</li>
```

Comment:
- [ ] It will be great to use a `<button>` tag for the remove button. This is because the `remove` should be a button not just a statement